src/components/BRadio/BRadio.md

```diff
@@ -0,0 +1,74 @@
+Example Radio Object
+
+Radios prop is an array of Radio Objects.
+
+| Key      | Required/Optional | Default | Description                              |
+| -------- | ----------------- | ------- | ---------------------------------------- |
+| id       | Required          |         | Needs to be unique in the radio group. Returned if radio button is selected.  |
+| value    | Required          |         | Label for radio button. |
+| disabled | Optional          | false   | Disables radio button.                   |
+
+Radio Button Options
+
+```js
+
+const radios = [
+  {id: 1, value:"Option" },
+  {id: 2, value:"Selected" },
+  {id: 3, value:"Disabled", disabled: true }
+]
+
+const radioCheckedId1 = 2
+
+<b-radio group-id="1" group-label="Radio Options" :radios="radios" v-model="radioCheckedId1"/>
```

Comment:
We need to enforce uniqueness on the group id vs the option ids. In this example (and the others) the radio group label isn't read because the radio option has id 1 so `aria-labelledby="1 radio-option-1"` just reads its own label.

Reply:
Since conditional validation really isn't a thing, I can think of two "solutions".
 
1. check and log an error if its not unique. 
2. Prefix group ids and option ids with some text, like "group" and "option".

Thoughts?

Reply:
I implemented option 1.